packages/blade/src/components/Button/Button/__tests__/Button.web.test.tsx

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
I've added some tests that seemed critical to me but not sure if what tests should be here since all of these(and more) are already covered on BaseButton